app/workspace/page.tsx

```
@@ -282,168 +597,344 @@ function WorkspaceContent() {
-              </div>
+          {/* Quick Actions */}
+          <GlassCard className="mb-8 p-4 sm:p-6">
+            <h2 className="text-lg sm:text-xl font-bold text-leather-100 mb-4">
+              Quick Actions
+            </h2>
+            <div className="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-6 gap-3">
+              <button
+                onClick={handleCreateDocument}
+                className="flex flex-col items-center justify-center p-4 rounded-lg bg-leather-900/30 hover:bg-leather-900/50 transition-all border border-leather-700/30 hover:border-leather-600"
+              >
+                <span className="text-3xl mb-2">📝</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)